pxr/usd/usdGeom/xformable.cpp

```diff
@@ -746,9 +746,9 @@ UsdGeomXformable::TransformMightBeTimeVarying() const
     if (opOrderVec.size() == 0) {
         return false;
     }
-
-    for (VtTokenArray::reverse_iterator it = opOrderVec.rbegin() ; 
-         it != opOrderVec.rend(); ++it) {
+    const VtTokenArray &opOrderVecConst = opOrderVec.AsConst();
+    for (VtTokenArray::const_reverse_iterator it = opOrderVecConst.rbegin() ; 
```

Comment:
I think you can just use `crbegin()` and `crend()` here.

Reply:
Ah, I see Matt commented the same on a subsequent instance, sorry for the repetition.